department-of-veterans-affairs/vets-website
# Add income receipt waivers

## Did you change site-wide styles, platform utilities or other infrastructure?

## Summary
Import Waiver of Receipt of Income - Section 12 into the Income & Asset Statement form (21P-0969)

## Issue
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/96712

## Associated Pull Request(s)
n/a

## How to run in local environment
1. Check out this branch locally
2. Run `vets-website` and `vets-api`
3. In your local environment, set the `income_and_assets_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/income_and_assets_form_enabled
4. Go to `http://localhost:3001/income-and-asset-statement-form-21p-0969/` in your browser

## How to verify
1. Go to chapter 12 - Income Receipt Waivers (/income-receipt-waivers-summary)
2. Verify that an discontinued income can be added, edited, and deleted
3. Verify that discontinued incomes are displayed on the 'Review' page

## What areas of the site does it impact?
Income & Asset Statement Application

## Screenshots
### Desktop
#### Summary Page
| No Waivers      | Waivers       |
| ----------- | ----------- |
|![localhost_3001_income-and-asset-statement-form-21p-0969_](https://github.com/user-attachments/assets/e74562d9-a0b6-487e-8e5b-8a49c6a8d5bc)|![localhost_3001_income-and-asset-statement-form-21p-0969_ (6)](https://github.com/user-attachments/assets/769a960f-c572-4115-8278-b649079ef5cf)


### Item Pages
| Relationship | Payer | Gross Amount |
| ----------- | ----------- | ----------- |
|![localhost_3001_income-and-asset-statement-form-21p-0969_ (1)](https://github.com/user-attachments/assets/5010d1c8-15e9-4f7a-9c1f-8c22b9e193ad)|![localhost_3001_income-and-asset-statement-form-21p-0969_ (2)](https://github.com/user-attachments/assets/5c81da3b-8510-49fb-8703-ff4819cd7ffe)|![localhost_3001_income-and-asset-statement-form-21p-0969_ (3)](https://github.com/user-attachments/assets/85b56c17-384f-4efa-bcc9-ef040172af55)



| Payments | Date | Expected Amount |
| ----------- | ----------- | ----------- |
|![localhost_3001_income-and-asset-statement-form-21p-0969_ (4)](https://github.com/user-attachments/assets/ae8ac7d5-3eb7-417a-b534-6a2ea09007a4)|![localhost_3001_income-and-asset-statement-form-21p-0969_ (7)](https://github.com/user-attachments/assets/1a01b337-893b-4f58-8592-fc71196b19dc)|![localhost_3001_income-and-asset-statement-form-21p-0969_ (5)](https://github.com/user-attachments/assets/d5bd6857-54b2-4207-9c65-ce0609c06cd1)




### Quality Assurance & Testing

### Error Handling

### Authentication

## Requested Feedback
(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
